### PR TITLE
Task 5：Standardize the comment of `func CheckRelease` in pkg/utils/helm/utils.go

### PR DESCRIPTION
**Pull request by heu-kevin**
### Ⅰ. Describe what this PR does
Standardize the comment of `func CheckRelease` in pkg/utils/helm/utils.go
mapreduce课程作业，完成Task 5，将pkg/utils/helm/utils.go中的方法`func CheckRelease`上的注释规范化

### Ⅱ. Does this pull request fix one issue?

NONE

### Ⅲ. List the added test cases (unit test/integration test) if any, please explain if no tests are needed.
Just fix comments

### Ⅳ. Describe how to verify it
NONE

### Ⅴ. Special notes for reviews
mapreduce课程作业，完成Task 5

### CODE REVIEW

File: pkg/utils/helm/utils.go [hunk 1]
```diff
@@ -64,9 +64,10 @@ func InstallRelease(name string, namespace string, valueFile string, chartName s
 	return err
 }
 
-/**
-* check if the release exist
- */
+
+// CheckRelease checks if the release derived from given name and namespace exist.
```
Comment (TrafalgarZZZ) [inline pkg/utils/helm/utils.go:68]: `derived from` looks a little bit weird to me
```suggestion
// CheckRelease checks if the release with the given name and namespace exist.
```

File: pkg/utils/helm/utils.go [hunk 2]
```diff
@@ -64,9 +64,10 @@ func InstallRelease(name string, namespace string, valueFile string, chartName s
...
+// It returns a bool variable exist and any error encountered.
+// If the release doesn't exist, the bool variable exist will be false.
```
Comment (TrafalgarZZZ) [inline pkg/utils/helm/utils.go:70]: This two lines are unnecessary. I think Line 68 is self-explained enough

**Review by TrafalgarZZZ — CHANGES_REQUESTED** [review]

File: pkg/utils/helm/utils.go (hunk 1 above)
Comment (heu-kevin) [reply pkg/utils/helm/utils.go]: I have already modify "derived from" to "with the"

File: pkg/utils/helm/utils.go (hunk 2 above)
Comment (heu-kevin) [reply pkg/utils/helm/utils.go]: I have deleted 69 and 70 lines.

**Review by TrafalgarZZZ — APPROVED** [review]
/lgtm

**Pull request merged**